ldk/go/Makefile

```diff
@@ -22,7 +22,20 @@ proto:
 	GO111MODULE=off go get -u github.com/golang/protobuf/protoc-gen-go
 	protoc -I ../../proto/ ../../proto/*.proto -I $(GOPATH)/src --go_out=plugins=grpc:proto/
 
-test:
+test-examples:
```

Comment:
Awesome! I was wondering if we should fire the example tests here. 👍

Reply:
Yeah, I didn't realize the tests examples tests were not already running. Some of them were quite out of date.